src/main/java/gregtech/common/tileentities/machines/multi/GT_MetaTileEntity_LargeBoiler.java

```diff
@@ -181,10 +181,30 @@ public boolean isCorrectMachinePart(ItemStack aStack) {
         return true;
     }
 
+    boolean isIntegratedCircuit(ItemStack input) {
+        return input != null && input.getUnlocalizedName().startsWith("gt.integrated_circuit");
+    }
```

Comment:
Do not reinvent the wheel, use this already built-in method instead:
```suggestion
    boolean isIntegratedCircuit(ItemStack input) {
        return Circuit_Integrated.isStackEqual(input, true, true);
    }
```
But then this single-use utility method could be:
- marked `private`: because it is only for internal usage,
- and marked `static`: because it has no side-effect on the instance state,
- or Inlined: because the single internal usage of a simple one-step condition.